frontend/src/metabase-types/api/cloud-migration.ts

```diff
@@ -0,0 +1,18 @@
+export type CloudMigrationState =
+  | "init"
+  | "setup"
+  | "dump"
+  | "upload"
+  | "cancelled"
```

Comment:
```suggestion
  | "canceled"
```

Reply:
So this is correct / matches the backend. @filipesilva might be best for us at some point to change to a word like stopped or something.